src/atomate2/vasp/flows/mp.py

```diff
@@ -0,0 +1,223 @@
+"""
+Module defining flows for Materials Project workflows.
+
+Reference: https://doi.org/10.1103/PhysRevMaterials.6.013801
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Sequence
+
+from jobflow import Flow, Maker
+
+from monty.serialization import loadfn
+from pkg_resources import resource_filename
+
+from atomate2.vasp.jobs.mp import (
+    MPGGARelaxMaker,
+    MPGGAStaticMaker,
+    MPMetaGGARelaxMaker,
+    MPMetaGGAStaticMaker,
+    MPPreRelaxMaker,
+)
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from jobflow import Job
+    from pymatgen.core.structure import Structure
+
+    from atomate2.vasp.jobs.base import BaseVaspMaker
+
+
+__all__ = ["MPGGARelax", "MPGGADoubleRelaxMaker", "MPMetaGGARelax", "MPMetaGGADoubleRelaxMaker"]
+
+
+@dataclass
+class _MPGenericThreeStep(Maker):
```

Comment:
Personally, I would be against defining the flows like this. It is quite complicated for users trying to understand what is going on. I don't feel that there is much boilerplate in a DoubleRelax + Static Maker.

Reply:
I've rewritten these: the hidden generic maker has been replaced by MPGGADoubleRelaxMaker. This maker features an optional final static, which is disabled by default. The other flows (MPGGARelax, MPMetaGGARelax, and MPMetaGGADoubleRelaxMaker) all inherit from MPGGADoubleRelaxMaker